docs/3_NEW_STACK.md

```diff
@@ -0,0 +1,13 @@
+# 3. Creating a New Stack
+
```

Comment:
It would be nice if every page had a link back to the quick start guide.

```suggestion
*Have you completed the quick start guide? Deploy your first infrastructure as code project with Cobalt by following the [quick-start guide](./2_QUICK_START_GUIDE.md).*
```